help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -224,3 +235,18 @@ To fix this error, verify that the **[!UICONTROL Mapping ID]** you provided in [
 Additional useful information from the [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/) is below:
 * [IOrganizationService.Update(Entity) Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.xrm.sdk.iorganizationservice.update?view=dataverse-sdk-latest)
 * [Update and delete table rows using the Web API](https://docs.microsoft.com/en-us/power-apps/developer/data-platform/webapi/update-delete-entities-using-web-api#basic-update)
+
+### Changelog
+
+This section captures the functionality and significant documentation updates made to this destination connector.
+
++++ View changelog
+
+|Release month|Update type|Description|
+|---|---|---|
+|August 2023|Functionality update| We fixed an issue where the user is unable to leverage a [!DNL Dynamics 365] custom field if it is not created within the default solution. An input field, **[!UICONTROL Customization Prefix]** has been introduced in the [Fill in destination details](#destination-details) step where the user would input the required prefix. (PLATIR-31602) |
```

Comment:
This is both a functionality AND a documentation update, so please update that section too.

Also, let's rephrase and split this update to be easier to understand:

- Fixed an issue where users were unable to use a [!DNL Dynamics 365] custom field if it was not created within the default solution.
- A new input field, **[!UICONTROL Customization Prefix]**, has been added in the [Fill in destination details](#destination-details) step, where users must input the required prefix.

Also, I don't really understand if these two updates are related to each other.

- What does "default solution" mean?
- What does the prefix do? 
- Is there a link between the prefix and the default solution or custom field? 

Please review this section and add some clarity around what this means for the customers.

Reply:
Updated the changelog.  The queries you raise are clarified by going through _Create field within [!DNL Dynamics 365] {#prerequisites-custom-field}. Terminology is specific to MSD. To elaborate again would be replicating their writeup.

Reply:
Let's rephrase this to be even clearer:

- Added support for [!DNL Dynamics 365] custom field prefixes. A new input field, **[!UICONTROL Customization Prefix]**, has been added in the [Fill in destination details](#destination-details) step.
- Fixed an issue where users were unable to use a custom field if it was not created within the default solution in [!DNL Dynamics 365].

Reply:
Both bullets are the same have changed to 

Added support for [!DNL Dynamics 365] custom field prefixes for custom fields which were not created within the default solution in [!DNL Dynamics 365]. A new input field, **[!UICONTROL Customization Prefix]**, has been added in the [Fill in destination details](#destination-details) step. (PLATIR-31602).